assembly/pom.xml

```diff
@@ -174,63 +174,19 @@
                 <descriptor>
                   mvn:org.apache.jclouds.karaf/jclouds-karaf/${jclouds.karaf.version}/xml/features
                 </descriptor>
+                <descriptor>
+                  mvn:org.apache.jclouds.karaf/jclouds-karaf-labs/${jclouds.karaf.version}/xml/features
+                </descriptor>
               </descriptors>
               <features>
                 <feature>config</feature>
-                <feature>jclouds</feature>
-                <feature>jclouds-commands</feature>
-                <feature>jclouds-compute</feature>
-                <feature>jclouds-guice</feature>
```

Comment:
> It's like maven transitive features, don't need to mention them if we don't directly depend on them.

Thanks for clarifying. I suspected as much, but I think we may be specifying similar features elsewhere - not sure why, perhaps for clarity?

But it's a minor comment - just wondering what the most consistent approach for this refactoring is, that's all.